### PR TITLE
Add svvy.sh to readme

### PR DESCRIPTION
**Pull request by roninjin10**
Good use of whatsabi imo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,6 +156,7 @@ console.log("Resolved to:", address);
 * [abi.w1nt3r.xyz](https://abi.w1nt3r.xyz/) - A frontend for whatsabi by [@w1nt3r_eth](https://twitter.com/w1nt3r_eth) - https://github.com/w1nt3r-eth/abi.w1nt3r.xyz
 * [ethcmd.com](https://www.ethcmd.com/) - Contract explorer frontend, [uses whatsabi for unverified contracts](https://github.com/verynifty/ethcmd)
 * [monobase.xyz](https://monobase.xyz) - Universal frontend, [uses whatsabi for unverified contracts](https://twitter.com/nazar_ilamanov/status/1659648915195707392)
+* [savvy](https://svvy.sh/) - A contract executor ui that uses whatsabi to fetch and execute cintracts against an in browser devnet
```
Comment (roninjin10) [inline README.md:159]:
```suggestion
* [savvy](https://svvy.sh/) - A contract executor ui that uses whatsabi to fetch abis and execute contract methods against an in browser devnet built by [0xpolarzero](https://x.com/0xpolarzero?s=21&t=ZtDaR85ib6evdD45FLpEmQ)
```

**Comment by shazow** [comment]
Are you sure they use whatsabi? Didn't find any mention here: https://github.com/0xpolarzero/savvy

**Comment by shazow** [comment]
Ah nevermind, github search was being silly https://github.com/0xpolarzero/savvy/blob/main/package.json#L43

**Comment by shazow** [comment]
Thanks for this! Good find. :) 

cc @0xpolarzero

**Comment by roninjin10** [comment]
Yea whatsabi is what pulls the abi when you enter an address here 
image

**Pull request merged**